### DISCUSSION TITLE
Weird Satin column

### DISCUSSION THREAD

**Discussion by traverseda** [inkstitch/inkstitch#1596] [Q&A] [answered]
So I'm trying to set up a rung style satin column from an existing shape. It *seems* like I've done everything right, but clearly I'm not getting the right results.

![image](https://user-images.githubusercontent.com/2125828/156888211-09bdc6f1-34b9-4be2-8246-c8e3d4b83b45.png)

When I reproduce roughly the same shape manually the satin column works.

![image](https://user-images.githubusercontent.com/2125828/156888501-49b5e0f0-607a-4ac2-bb19-9fece720320f.png)

Any idea what's going on?

**Comment by tyrosinase** [comment, marked as answer]
The rails are going in opposite directions, which is great for string art but not so great for satin. If you use the node tool and click on one of the rails so that one or more of its nodes are highlighted, `Path > Reverse` will reverse only that segment. (If that makes the whole thing go the wrong direction, go to the regular selector and select the object and Reverse it all.)

If you go into the preferences somewhere, there's a way to add little arrow indicators to the highlight line when you're in node-editing mode. It's absolutely invaluable to see which direction the stitches are gonna go on both running and satin stitches.

**Reply by traverseda** [reply]
Thanks, that did the trick.
